bigframes/ml/compose.py

```diff
@@ -147,10 +254,30 @@ def camel_to_snake(name):
 
                     found_transformer = True
                     break
+
             if not found_transformer:
-                raise NotImplementedError(
-                    f"Unsupported transformer type. {constants.FEEDBACK_LINK}"
+                transformer_cls = CustomTransformer.find_matching_transformer(
+                    transform_sql
                 )
+                if transformer_cls:
+                    transformers_set.add(
+                        (
+                            camel_to_snake(transformer_cls.__name__),
+                            *transformer_cls._parse_from_sql(transform_sql),  # type: ignore
+                        )
+                    )
+                    found_transformer = True
+
+            if not found_transformer:
+                if not transform_sql.startswith("ML.") and "/*CT." not in transform_sql:
+                    continue  # ignore other patterns, only report unhandled known patterns
+                if transform_sql.startswith("ML."):
+                    raise NotImplementedError(
+                        f"Unsupported transformer type. {constants.FEEDBACK_LINK}"
+                    )
+                raise ValueError("Missing custom transformer")
```

Comment:
Rather than ValueError and NotImplementedError, could we provide a general purpose UnknownTransformer class? I'm aware this would lose some transparency, but maybe it could include an attribute that shows the SQL substring it corresponds to?

Reply:
> Rather than ValueError and NotImplementedError, 
> could we provide a general purpose UnknownTransformer class? [...]

The `NotImplementedError` was not changed from the original code.
So, I would leave it as it is.
And the ValueError should not happen any more, 
because any not `ML.xxx` sql will be translated into an `SQLScalarColumnTransformer`